LunaDll/LuaMain/LuaProxyFFI.cpp

```diff
@@ -739,6 +739,51 @@ typedef struct ExtendedBlockFields_\
     {
         return {gMouseHandler.GetX(), gMouseHandler.GetY()};
     }
+	FFI_EXPORT(void) LunaLuaSetFullscreen(bool enable)
+    {
+        // Toggling fullscreen without maximizing the window/double clicking the window.
+		if (gMainWindowHwnd != NULL)
+        {
+            WINDOWPLACEMENT wndpl;
+            wndpl.length = sizeof(WINDOWPLACEMENT);
+            if (GetWindowPlacement(gMainWindowHwnd, &wndpl))
+            {
+                if ((wndpl.showCmd == SW_MAXIMIZE) && (enable == false))
+                {
+                    ShowWindow(gMainWindowHwnd, SW_RESTORE);
+                } else {
+					if (enable == true) {
+						ShowWindow(gMainWindowHwnd, SW_MAXIMIZE);
+					}
+                }
+            }
+        }
+    }
+	FFI_EXPORT(void) LunaLuaToggleWindowFocus(bool enable)
+    {
+        // Setting the focus to the window
+		if (enable) {
+			gStartupSettings.runWhenUnfocused = true;
+        } else {
+			gStartupSettings.runWhenUnfocused = false;
+        }
```

Comment:
This still doesn't address what I noted in an earlier comment, where if you call this with `false`, it won't update the current state based on window focus. 

To repeat the relevant bit I said: 
> The best fix to this is changing the way `gStartupSettings.runWhenUnfocused` works, so it doesn't prevent setting `gMainWindowFocused` to false, but instead the places that currently check `gMainWindowFocused` should probably check `gMainWindowFocused || gStartupSettings.runWhenUnfocused`.